content/docs/cucumber/state.md

```diff
@@ -476,12 +529,14 @@ With Rails, you can also turn off transaction globally in your `features/support
 ```
 Cucumber::Rails::World.use_transactional_fixtures = false
 ```
+{{% /block %}}
```

Comment:
In the Java and Kotlin tabs, there is this statement: "Ruby tools provide specific ways to turn of transactions." - why does it say anything about Ruby in those?

Reply:
Again, because the headers will be visible, but the content will not be relevant for those languages.